src/workers/postDeletedSharedPostCleanup.ts

```diff
@@ -26,20 +38,42 @@ const worker: Worker = {
       );
     }
 
-    await con
-      .getRepository(SharePost)
-      .createQueryBuilder()
-      .update()
-      .where({
-        sharedPostId: post.id,
-      })
-      .set({
-        showOnFeed: false,
-        flags: updateFlagsStatement<Post>({
+    await Promise.all([
+      await con
+        .getRepository(SharePost)
+        .createQueryBuilder()
+        .update()
+        .where({
+          sharedPostId: post.id,
```

Comment:
Make sure we have index for sharedPostId

Reply:
We do have one:
IDX_sharedPostId (it's just singular, but for this query probably ok)